requirements.txt

```diff
@@ -16,4 +16,19 @@ selenium
 webdriver-manager
 
 #gunicorn package to run web app on Heroku
-gunicorn
\ No newline at end of file
+gunicorn
+
+ # Reference: https://pypi.org/project/webdriver-manager/
+webdriver-manager
```

Comment:
Please consider removing duplicate entries in requirements file.  For instance, webdriver-manager, gunicorn, etc are listed twice.